mmv1/templates/terraform/examples/cloud_run_service_multiple_regions.tf

```diff
@@ -0,0 +1,230 @@
+
+  # Use an explicit depends_on clause to wait until API is enabled
+  depends_on = [
+    google_project_service.compute_api
+  ]
+}
+
+resource "google_compute_backend_service" "lb_default" {
+  name                  = "myservice-backend"
+  project               = data.google_project.app.project_id
+  load_balancing_scheme = "EXTERNAL_MANAGED"
+
+  backend {
+    balancing_mode  = "UTILIZATION"
+    capacity_scaler = 0.85 # TODO what does this do? docs are unclear! :(
+    group           = google_compute_region_network_endpoint_group.lb_default[0].id
+  }
+
+  backend {
+    balancing_mode  = "UTILIZATION"
+    capacity_scaler = 0.85 # TODO what does this do? docs are unclear! :(
+    group           = google_compute_region_network_endpoint_group.lb_default[1].id
+  }
+
+  # Use an explicit depends_on clause to wait until API is enabled
+  depends_on = [
+    google_project_service.compute_api,
+  ]
+}
+
+
+resource "google_compute_url_map" "lb_default" {
+  name            = "myservice-lb-urlmap"
+  project         = data.google_project.app.project_id
+  default_service = google_compute_backend_service.lb_default.id
+
+  path_matcher {
+    name            = "allpaths"
+    default_service = google_compute_backend_service.lb_default.id
+    route_rules {
+      priority = 1
+      url_redirect {
+        https_redirect         = true
+        redirect_response_code = "MOVED_PERMANENTLY_DEFAULT"
+      }
+    }
+  }
+}
+
+resource "google_compute_managed_ssl_certificate" "lb_default" {
+  name    = "myservice-ssl-cert"
+  project = data.google_project.app.project_id
+
+  managed {
+    domains = [var.domain_name]
+  }
+}
+
+resource "google_compute_target_https_proxy" "lb_default" {
+  name    = "myservice-https-proxy"
+  project = data.google_project.app.project_id
+  url_map = google_compute_url_map.lb_default.id
+  ssl_certificates = [
+    google_compute_managed_ssl_certificate.lb_default.name
+  ]
+  depends_on = [
+    google_compute_managed_ssl_certificate.lb_default
+  ]
+}
+
+resource "google_compute_global_forwarding_rule" "lb_default" {
+  name                  = "myservice-lb-forwarding-rule"
+  project               = data.google_project.app.project_id
+  load_balancing_scheme = "EXTERNAL_MANAGED"
+  target                = google_compute_target_https_proxy.lb_default.id
+  ip_address            = google_compute_global_address.lb_default.id
+  port_range            = "443"
+  depends_on            = [google_compute_target_https_proxy.lb_default]
+}
+
+resource "google_compute_region_network_endpoint_group" "lb_default" {
+  count                 = length(var.run_regions)
+  project               = data.google_project.app.project_id
+  name                  = "myservice-neg"
+  network_endpoint_type = "SERVERLESS"
+  region                = var.run_regions[count.index]
+  cloud_run {
+    service = google_cloud_run_service.run_default[count.index].name
+  }
+}
+
+output "load_balancer_ip_addr" {
```

Comment:
is the output necessary for this example?

Reply:
Sort of - the end-user will need to enter this value into their DNS configuration settings.